src/components/common/NetworkSelector/NetworkMultiSelector.tsx

```diff
@@ -35,6 +42,20 @@ const NetworkMultiSelector = ({ name }: { name: string }): ReactElement => {
     router.push(networkLink)
   }
 
+  const isOptionDisabled = (chain: ChainInfo) => {
```

Comment:
Instead of hardcoding zkSync to be excluded we should check the `safe-deployments` and check that they have a canonical address deployed for version 1.4.1 .

Here is a snippet that does that would be this for instance:
```ts
  const hasCanonicalSingletonDeployment = Boolean(
    getSafeSingletonDeployment({ version: data.safeVersion })?.deployments.canonical,
  )
```

The reason is that there could always be chains that do not have 1.4.1 support yet or that do not share the canonical deployment.